cmd/plasmacli/subcmd/query/status.go

```diff
@@ -0,0 +1,32 @@
+package query
+
+import (
+	"fmt"
+	"github.com/FourthState/plasma-mvp-sidechain/client"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/spf13/cobra"
+)
+
+// StatusCmd returns the current state of the eth connection (syncing, crashed etc)
```

Comment:
Not the right godoc. Think this godoc is meant for the other status command

Reply:
What should it be?

Reply:
this checks the current block number